spec/dummy/config/locales/cy.yml

```diff
@@ -0,0 +1,2 @@
+cy:
```

Comment:
There is, but I guess this is the easiest approach in this case. Ignore that comment :smile: